### DISCUSSION TITLE
UI Feedback - Subscribers List

### DISCUSSION THREAD

**Discussion by sensingturtle** [buttondown/discussions#53] [Ideas]
### Current Behaviour (see video):

https://github.com/user-attachments/assets/87320b89-30d4-4247-a0d4-b88ff88c856c

### Expected Behaviour:

When clicking on the subscriber’s email (i.e. `person1@xyz.com`), I expect their checkbox to not get ticked because my goal (in this interaction’s context) is to just look at more details about this particular subscriber and not take any action like `Ban` or `Delete` (that appears in the toolbar below when the checkbox is toggled on).

Basically, clicking on the subscriber’s email (i.e. `person1@xyz.com`) should only toggle the visibility of the details sidebar (that is on the right) and not its corresponding checkbox. Just like clicking on the subscriber’s checkbox only toggles the visibility of the toolbar below.

Please correct me if the current behavior is intentional.

**Comment by jmduke** [comment]
Huh yeah, this is not ideal. We probably are missing some sort of event capture here. @mary-ext mind poking around?
